src/qt/intro.cpp

```diff
@@ -130,7 +130,7 @@ Intro::Intro(QWidget *parent, uint64_t blockchain_size, uint64_t chain_state_siz
         .arg(tr(PACKAGE_NAME))
         .arg(m_blockchain_size)
         .arg(2009)
```

Comment:
this should be 2018 which is when the liquidv1 chain started

Reply:
I updated it to 2016 (the start of Elements Alpha) in an extra commit.